library/Frontend/src/Juvix/Frontend/Sexp.hs

```diff
@@ -185,6 +187,33 @@ transGuardBody _alsee trans (Types.Body b) = trans b
 transGuardBody False trans (Types.Guard c) = transCond trans c
 transGuardBody True transs (Types.Guard c) = transCondMultiple transs c
 
+--------------------------------------------------------------------------------
+-- Effect Expansion
+--------------------------------------------------------------------------------
+
+transHand :: Types.Handler -> Sexp.T
+transHand (Types.Hand name ops ret) =
+  Sexp.list
+    [ Sexp.atom ":defhandler",
+      Sexp.atom (NameSymbol.fromSymbol name),
+      Sexp.listStar [Sexp.atom ":ops", Sexp.list (fmap transOperation ops)],
+      transOperation ret
+    ]
+
+transEffect :: Types.Effect -> Sexp.T
+transEffect Types.Eff {effName, effOps, effRet} =
+  Sexp.list
+    [ Sexp.atom ":defeff",
+      Sexp.atom (NameSymbol.fromSymbol effName),
+      Sexp.listStar [Sexp.atom ":ops", Sexp.list (fmap transSig effOps)],
```

Comment:
Same here.

would make sense if the given form doens't know but here it seems always fixed, or if the form was in an style like

```juvix
(test :a 3 :b 4)
```
but in this case it's not?

Reply:
Yeah I'm not sure this comes out as you expect it to, as now upon reasoning about it you want something like 

```lisp
(:defhandler print
  (:defops print (x) (print x))
  (:defops fud (x) (print x))
  (:defun return (x) (toString x)))
```

right?

Reply:
no, return is also an operation, but a special one.